app/helpers/enterprises_helper.rb

```diff
@@ -20,6 +20,27 @@ def available_shipping_methods
     shipping_methods.uniq
   end
 
+  def product_import_dates
+    import_dates = Spree::Variant.
+      select('spree_variants.import_date').
+      joins(:product).
+      where('spree_products.supplier_id IN (?)
+      AND spree_variants.is_master = false
+      AND spree_variants.import_date IS NOT NULL
+      AND spree_variants.deleted_at IS NULL', editable_enterprises.collect(&:id))
+
+    import_dates.uniq.collect(&:import_date).sort.reverse
+  end
+
+  def inventory_import_dates
+    import_dates = VariantOverride.
+      select('variant_overrides.import_date').
+      where('variant_overrides.hub_id IN (?)
+      AND variant_overrides.import_date IS NOT NULL', editable_enterprises.collect(&:id))
+
+    import_dates.uniq.collect(&:import_date).sort.reverse
+  end
+
```

Comment:
I'd rather implement this in a new service object rather than adding yet another responsibility to an already general-purpose module such as `EnterpriseHelper`. There's no problem on having explicit calls to an object on the controllers that use these methods.